examples/README.md

```diff
@@ -70,3 +70,4 @@ In addition, integration modules are available for the following libraries, prov
 ### External libraries using Optuna
 
 * [Analyzing optimization results using Neptune](https://neptune-contrib.readthedocs.io/user_guide/monitoring/optuna.html)
+* [A scikit-learn compatible LightGBM estimator with Optuna](https://github.com/Y-oHr-N/OptGBM)
```

Comment:
How about including the package name to the text as follows?
```suggestion
* [OptGBM: A scikit-learn compatible LightGBM estimator with Optuna](https://github.com/Y-oHr-N/OptGBM)
```

Reply:
Looks better. Thanks!